osmosis-1/osmosis.zone_assets.json

```diff
@@ -2307,6 +2307,13 @@
       "path": "transfer/channel-642/factory/migaloo1etlu2h30tjvv8rfa4fwdc43c92f6ul5w9acxzk/uguppy",
       "osmosis_verified": false,
       "osmosis_unlisted": true
+    },
+    {
+      "chain_name": "haqq",
+      "base_denom": "aISLM",
+      "path": "transfer/channel-1575/aISLM",
```

Comment:
This IBC connection is not yet registered to the Chain Registry. Please register it there https://github.com/cosmos/chain-registry/tree/master/_IBC